package.json

```diff
@@ -95,11 +102,15 @@
     "eslint-plugin-react": "^7.29.4",
     "eslint-plugin-react-hooks": "^4.3.0",
     "jest": "^27.5.1",
+    "relative-deps": "^1.0.7",
     "typescript": "^4.1.2"
   },
   "engines": {
     "node": ">=14"
   },
   "sideEffects": false,
-  "license": "MIT"
+  "license": "MIT",
+  "relativeDependencies": {
```

Comment:
is this also a new thing from this PR?

Reply:
Yes! Will add docs about relative dependencies and how it works today and put it in this MR